packages/foreman/foreman/foreman.spec

```diff
@@ -158,13 +159,15 @@ BuildRequires: %{?scl_prefix_ruby}rubygem(rake) >= 0.8.3
 BuildRequires: %{?scl_prefix_ruby}rubygem(rdoc)
 BuildRequires: %{?scl_prefix}rubygem(bundler_ext)
 # start specfile sqlite BuildRequires
-BuildRequires: %{?scl_prefix}rubygem(sqlite3) >= 1.3.6
-BuildRequires: %{?scl_prefix}rubygem(sqlite3) < 1.4.0
+BuildRequires: %{?scl_prefix}rubygem(sqlite3) >= 1.4
+BuildRequires: %{?scl_prefix}rubygem(sqlite3) < 2.0
 # end specfile sqlite BuildRequires
 
 
 # start specfile main BuildRequires
-BuildRequires: %{?scl_prefix}rubygem(rails) = 5.2.1
+BuildRequires: %{?scl_prefix}rubygem(cgi) >= 0.a
+BuildRequires: %{?scl_prefix}rubygem(yaml) >= 0.a
```

Comment:
Where did these come from?

Reply:
The script uses rubygems and internally that also calls the `gem()` method. Perhaps Ruby loads these internally